src/middlewared/middlewared/plugins/interface/configure.py

```diff
@@ -165,13 +114,15 @@ def configure(self, data, aliases, options):
             try:
```

Comment:
Our Linux netif implementation does not manage descriptions, we might just remove this (keep in API for user's notes)

Reply:
Good point. I'll make another ticket/PR.